ntpcheck/cmd/utils.go

```diff
@@ -259,7 +310,25 @@ var printLeapCmd = &cobra.Command{
 	Short: "Prints leap second information from the system timezone database",
 	Run: func(cmd *cobra.Command, args []string) {
 		ConfigureVerbosity()
-		if err := printLeap(); err != nil {
+		if err := printLeap(sourceLeapSeconds); err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+	},
+}
+
+var addFakeSecondZoneInfoCmd = &cobra.Command{
+	Use:   "addfakesecond",
+	Short: "Adds fake second to zoneinfo database",
+	Long:  `'addfakesecond' will read current zoneinfo leap second file from srcfile, add fake second
```

Comment:
Alternatively wouldn't allowing to specify UTC timestamp be better?

Reply:
Do you mean providing UTC string of timestamp for new leap second? Or how do you want to provide it?

Reply:
Yeah maybe just let user to specify ANY timestamp not only EOM/EOY. But just as an idea.

Reply:
maybe next time

Reply:
changed to 'offset in Months' as Alex suggested